tests/spec/core/lint-hrefs-ids-spec.js

```diff
@@ -0,0 +1,43 @@
+"use strict";
+describe("Core — Lint href's matching id's", function() {
+  afterAll(function(done) {
```

Comment:
No luck with that, you can check out my attempts [here](https://github.com/futpib/respec/commit/8b60c98b88fdce620b144a10c4739f20400e0831). Here is the error I got:

```
  Core — Lint href's matching id's
    ✖ encountered a declaration exception
      Chrome 59.0.3071 (Linux 0.0.0)
    Error: it expects a function argument; received [object AsyncFunction]
        at Suite.describe (tests/spec/core/lint-hrefs-ids-spec.js:8:3)
        at tests/spec/core/lint-hrefs-ids-spec.js:3:1
```